src/validation.cairo

```diff
@@ -30,8 +30,14 @@ fn validate_prev_block_hash(self: @ChainState, block: @Block) -> Result<(), Byte
 }
 
 fn validate_proof_of_work(self: @ChainState, block: @Block) -> Result<(), ByteArray> {
-    // TODO: implement
-    Result::Ok(())
+    let target_u256: u256 = block.header.bits.clone().into();
```

Comment:
you need to use bit_to_target function. It is not yet finished: https://github.com/keep-starknet-strange/raito/pull/15

Reply:
for now you can change validate_proof_of_work to accept `target` instead of `ChainState`.